dist/metricsgraphics.js

```diff
@@ -4660,7 +4660,12 @@ MG.button_layout = function(target) {
   }
 
   function mg_default_color_for_path(this_path, line_id) {
-    this_path.classed('mg-line' + (line_id) + '-color', true);
+      if(line_id <= 10)
+
+          this_path.classed('mg-line' + (line_id) + '-color', true);
+
+      else
+          this_path.classed('mg-line' + '-color', true);
```

Comment:
With this approach, lines of a greater number than 10 will be always be black, which isn't quite what we want. I think the right approach here is to give each line both the mg-line-color AND the mg-line-X-color. That gives the user the opportunity to extend the set of supported colors while still defaulting to black when that isn't present.

Reply:
Yeah you are right. I'll implement the same.